crates/symbolicli/src/main.rs

```diff
@@ -0,0 +1,655 @@
+        let mut row = Row::empty();
+        let is_inline = Some(frame.raw.instruction_addr)
+            == frames
+                .peek()
+                .map(|next_frame| next_frame.raw.instruction_addr);
+
+        let trust = if is_inline {
+            "inline"
+        } else {
+            match frame.raw.trust {
+                FrameTrust::None => "none",
+                FrameTrust::Scan => "scan",
+                FrameTrust::CfiScan => "cfiscan",
+                FrameTrust::Fp => "fp",
+                FrameTrust::Cfi => "cfi",
+                FrameTrust::PreWalked => "prewalked",
+                FrameTrust::Context => "context",
+            }
+        };
+
+        let instruction_addr = frame.raw.instruction_addr.0;
+
+        row.add_cell(cell!(trust));
+        row.add_cell(cell!(r->format!("{instruction_addr:#x}")));
+
+        match frame.raw.package {
+            Some(module_file) => {
+                let module_addr = module_addr_by_code_file[&module_file].raw.image_addr.0;
+                let module_file = split_path(&module_file).1;
+                let module_rel_addr = instruction_addr - module_addr;
+
+                row.add_cell(cell!(module_file));
+                row.add_cell(cell!(r->format!("+{module_rel_addr:#x}")));
+            }
+            None => row.add_cell(cell!("").with_hspan(2)),
+        }
+
+        match frame.raw.function.or(frame.raw.symbol) {
+            Some(func) => {
+                let sym_rel_addr = frame
+                    .raw
+                    .sym_addr
+                    .map(|sym_addr| format!(" +{:#x}", instruction_addr - sym_addr.0))
+                    .unwrap_or_default();
+
+                row.add_cell(cell!(func));
+                row.add_cell(cell!(r->sym_rel_addr));
+            }
+            None => row.add_cell(cell!("").with_hspan(2)),
+        }
+
+        if let Some(file) = frame.raw.filename {
+            let line = frame.raw.lineno.unwrap_or(0);
+
+            row.add_cell(cell!(format!("{file}:{line}")));
+        }
+
+        table.add_row(row);
+    }
+
+    table.printstd();
+}
+
+fn print_pretty(mut response: CompletedSymbolicationResponse) {
+    let module_addr_by_code_file: HashMap<_, _> = response
+        .modules
+        .into_iter()
+        .filter_map(|module| Some((module.raw.code_file.clone()?, module)))
+        .collect();
+
+    if response.stacktraces.is_empty() {
+        return;
+    }
+
+    let crashing_thread_idx = response
+        .stacktraces
+        .iter()
+        .position(|s| s.is_requesting.unwrap_or(false))
+        .unwrap_or(0);
+
+    let thread = response.stacktraces.swap_remove(crashing_thread_idx);
+
+    let mut table = Table::new();
+    table.set_format(*prettytable::format::consts::FORMAT_CLEAN);
+    let mut frames = thread.frames.into_iter().enumerate().peekable();
+    while let Some((i, frame)) = frames.next() {
+        let is_inline = Some(frame.raw.instruction_addr)
```

Comment:
Same goes for most of the values extracted from the data. They repeat for pretty and compact, with a small details modified